easybuild/easyconfigs/n/nullarbor/nullarbor-2.0.20191013.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	GPLv2.0
+#
+# Notes::	This is a conda version installation
+##
+
+easyblock = 'Conda'
+
+name = 'nullarbor'
+version = '2.0.20191013'
+
+homepage = 'https://github.com/tseemann/nullarbor'
+description = """
+Pipeline to generate complete public health microbiology reports from sequenced isolates
+"""
+
+toolchain = SYSTEM
+
+channels = ['conda-forge', 'bioconda']
+
+requirements = '%(name)s=%(version)s python=3.6'
+
+dependencies = [
+    ('Miniconda3', '4.7.10', '', True)
+]
+
+sanity_check_commands = [('nullarbor.pl', '-h')]
```

Comment:
there is `nullarbor.pl --check`, but if fails with missing databases, should those also be installed?

Reply:
@migueldiascosta Thx for pointing `--check` option. I think it's a duplicate feature as `sanity_check` checks dir `perl5` which checks `perl` modules,  `files` checks all binaries. 
You probably are right, a more explicit option can be invoked to check each required `perl` modules. 

A complete `nullarbor` has two parts `framework` and `dataset`, this easyconfig only cater `framework` part. Given the reference `database` is of a reasonable size `24G` altogether and different research may use different reference databases, I would prefer to leave this part to the individual institution. I can append a few lines of instructions at the end, see what you @migueldiascosta think.